esmvaltool/recipes/examples/recipe_check_obs.yml

```diff
@@ -212,7 +212,7 @@ diagnostics:
       chl:
     additional_datasets:
       - {dataset: ESACCI-OC, project: OBS, mip: Omon, tier: 2,
-         type: sat, version: fv3.1, start_year: 1997, end_year: 2016}
+         type: sat, version: fv5.0, start_year: 1998, end_year: 2020}
```

Comment:
Just taking a quick look at the cmorizer to see how to integrate that into #1657. I noticed that the entry in the recipe_check_obs contains a typo. Could you please change `OBS` to `OBS6`?